CHANGELOG-0.x.md

```diff
@@ -1,3 +1,10 @@
+# v0.7.1
+
+### New features
+
+### Misc.
+* Release 0.7.0 part 3/3 ([#224](https://github.com/kubernetes-sigs/aws-fsx-csi-driver/pull/224), [@khoang98](https://github.com/khoang98))
```

Comment:
probably we can remove this and just say 'updating to latest al2 base image ' or something like that

Reply:
Sounds good - going to close this PR and make this change in a different one.